### PR TITLE
Update Go 1.13 to 1.16

### PR DESCRIPTION
**Pull request by yamatcha**
This reverts commit 298334eee406799de4c902076f2e5ba2cd3ff6ba.

### CODE REVIEW

**Comment by ymmt2005** [comment]
@yamatcha 
Also, update this line for Go 1.16:
```
necogcp-app/app.yaml:runtime: go113
```

**Review by yokaze — DISMISSED** [review]
LGTM

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -10,10 +10,8 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
-      # Cloud Functions supports only Go runtimes 1.11 and 1.13.
-      # Therefore, this test confirms that the source is able to build with Go 1.13.
       matrix:
-        go-version: ["1.13", "1.17"]
+        go-version: ["1.16", "1.17"]
```
Comment (masa213f) [inline .github/workflows/main.yaml:14]: Could you remain this comment?  
Without this comment, we don't know why testing with go 1.16.

```suggestion
      # Cloud Functions does not support the latest Go runtimes. Currently, it only supports up to Go 1.16.
      # Therefore, this test confirms that the source is able to build with the supported Go version.
      matrix:
        go-version: ["1.16", "1.17"]
```

**Review by masa213f — CHANGES_REQUESTED** [review]

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**